src/components/Photography/ExifOverlay.tsx

```diff
@@ -22,4 +28,28 @@ const ExifOverlay = styled.div`
   width: 100%;
 `
 
+interface Props {
+  image?: S3ImageAsset | any
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)